### ISSUE TITLE
Royal Saute unusable due to spelling error in SQL

### ISSUE THREAD

**Issue by RAIST5150** [DerpyProjectGroup/topaz#636]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: Everything Branch

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
discovered on Canaria server, running Everything Branch

Can craft the items just fine, can even craft the NQ into subs no problem--but cannot eat the Royal Saute.  Was told the name in the DB needs to match the file in globals/items, which they do not (note "saute" versus "sautee":

https://github.com/LandSandBoat/server/blob/base/scripts/globals/items/plate_of_royal_saute.lua

```lua
INSERT INTO `item_basic` VALUES (4295,0,'plate_of_royal_sautee','royal_sautee',1,1580,52,0,70);         -- https://www.bg-wiki.com/bg/Royal_Saute
```

https://github.com/LandSandBoat/server/blob/base/scripts/globals/items/plate_of_coeurl_saute.lua
```lua
INSERT INTO `item_basic` VALUES (4548,0,'plate_of_coeurl_saute','coeurl_saute',1,1580,52,0,628);        -- https://www.bg-wiki.com/bg/Coeurl_Saute
```

**Comment by RAIST5150** [comment]
Went in to try to make this edit tonight, but the file is rendering in raw format... no link to edit it or anything like I've seen on the smaller SQL files.  Not sure how to go about starting a PR for this?  The "download" button just opens it as normal.  I am missing something...

**Comment by TeoTwawki** [comment]
> 
> 
> Went in to try to make this edit tonight, but the file is rendering in raw format... no link to edit it or anything like I've seen on the smaller SQL files. Not sure how to go about starting a PR for this? The "download" button just opens it as normal. I am missing something...

oh dear. Do not use the website for this work friend, the website is a terrible way to work! But in case you ever really need to, you would open raw then save page as plain text then name the file to whatever. But seriously you need a git client. Then clone your fork locally to do the edits, push changes back to your fork's branch you are working off of (make new forks as you go!) and the changes will automatically be on github and attach to any existing pull request if one already exists for that branch. Much saner workflow.

If on windows I recommend [tortoisegit](https://tortoisegit.org/support/faq/#prerequisites). sending changes to github is then as easy as _Right Click -> Push_

If on linux you likely already have a command line git client.

**Comment by RAIST5150** [comment]
oof... that is my biggest problem I guess.  Never used any form of IDE. Installed Eclipse and other stuff on workstations for the guys to use... just never touched the tools myself.

Actually a hardware geek myself.  Literally... raw edits of websites/webapps (java/tomcat/ESRI) and such is how I got exposed to this stuff back in the day.  Even on the AS/400's, I would be looking at the flat files and relaying stuff to the dev's--never actually got to witness the sausage making on those systems.

IDK how you guys do what you do... but sure glad y'all understand it!

**Comment by TeoTwawki** [comment]
I don't think this counts as an IDE, but tortoise does get you a GUI for doing git things (and give is just version control):

![image](https://user-images.githubusercontent.com/6871475/139667280-123f2950-261a-496d-b511-a696b2386689.png)

**Comment by RAIST5150** [comment]
This should be addressed by PR #910 if it is accepted/merged

**Comment by RAIST5150** [comment]
Closed via #910 commit
